src/test/java/christmas/domain/EventProcessTest.java

```diff
@@ -0,0 +1,60 @@
+package christmas.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class EventProcessTest {
+    @DisplayName("이벤트에 충분한 전체 주문 금액으로 전체 혜택 도출")
+    @Test
+    void takeAllBenefitWithSufficientTotalOrderPrice() {
+        Date dateInput = new Date(3);
+        Order orderInput = new Order(setUpSufficientTotalOrderPriceOrder());
+        EventProcess eventProcess = new EventProcess();
+        EventResult eventResult = eventProcess.takeAllBenefit(dateInput, orderInput);
+        Map<BenefitType, Integer> result = eventResult.getAllBenefit();
+
+        assertThat(result.get(BenefitType.GIFT)).isEqualTo(25000);
+        assertThat(result.get(BenefitType.D_DAY)).isEqualTo(1200);
+        assertThat(result.get(BenefitType.WEEKDAY)).isEqualTo(2023);
+        assertThat(result.get(BenefitType.WEEKEND)).isEqualTo(0);
+        assertThat(result.get(BenefitType.SPECIAL)).isEqualTo(1000);
```

Comment:
(추천드려요)

assertThat만 사용하면 21번째 줄에서 실패했을 때 밑에 22~25번 라인까지 검증하지 않습니다!
반면 assertAll을 사용하면 모든 assertThat을 실행시키기 때문에 무엇이 실패했는지 확실하게 찾을 수 있는 장점이 있습니다!
```suggestion
          assertAll(
                () -> assertThat(result.get(BenefitType.GIFT)).isEqualTo(25000),
                () -> assertThat(result.get(BenefitType.D_DAY)).isEqualTo(1200),
                () -> assertThat(result.get(BenefitType.WEEKDAY)).isEqualTo(2023),
                () -> assertThat(result.get(BenefitType.WEEKEND)).isEqualTo(0),
                () -> assertThat(result.get(BenefitType.SPECIAL)).isEqualTo(1000)
        );
```

Reply:
와,, 확실하게 테스트 할 수 있겠네요 감사합니다!!